src/doc/functional/_enrollment.rst

```diff
@@ -2,124 +2,141 @@
 Enrollment Services
 -------------------
 
+This interface describes enrollment services in the context of an identity system. It is based on
+the following principles:
+- When enrollment is done in one step, the CreateEnrollment can contain all the data and an additional flag (finalize) to indicate every data was collected.
```

Comment:
Typo: missing empty line, needed by Sphinx/rst to recognize a bullet list is next.